Makefile

```diff
@@ -43,6 +42,22 @@ endif
 REGEX_STRING?=''
 REGEX_PATTERN?=''
 
+# Exclude these files from coverage reports
+define COVERAGE_EXCLUDED_FILES
+--exclude-files apps/trivium/src/trivium/* \
+--exclude-files apps/trivium/src/kreyvium/* \
+--exclude-files apps/trivium/src/static_deque/* \
+--exclude-files apps/trivium/src/trans_ciphering/* \
+--exclude-files tasks/src/* \
+--exclude-files tfhe/benches/boolean/* \
+--exclude-files tfhe/benches/core_crypto/* \
+--exclude-files tfhe/benches/shortint/* \
+--exclude-files tfhe/benches/integer/* \
+--exclude-files tfhe/benches/* \
+--exclude-files tfhe/examples/regex_engine/* \
+--exclude-files tfhe/examples/utilities/*
```

Comment:
does the command support `**` patterns ? i.e. recursive globbing ? this looks convoluted to exclude dirs and sub dirs

Reply:
I didn't tried recursive globing.

Reply:
Nope recursive don't work sadly, we need to list the subdirectories :disappointed:

Reply:
-_- this could deserve an issue on the tool page

Reply:
`      --lib                        Test only this package's library unit tests`

from their Readme could it be what we need ?

Reply:
let me know if lib improves things otherwise I'll approve

Reply:
Nope it doesn't exclude files.